### PR TITLE
Updates public repository with latest functionality

### PR DESCRIPTION
**Pull request by raoulsullivan**
The functionality used on the bank referral scheme is untouched

### CODE REVIEW

File: sme_finance_application_schema/batch_response_v1 [hunk 1]
```diff
@@ -1,9 +1,15 @@
 {
     "$schema":"http://json-schema.org/draft-04/schema#",
+    "id": "https://www.fundingoptions.com/schema/batch_response_v1",
     "title":"Finance Application Batch response",
     "description":"The response object for receipt of an SME Finance Application Batch",
     "additionalProperties": false,
     "type": "object",
+    "definitions":{
```
Comment (cybojenix) [inline sme_finance_application_schema/batch_response_v1]: definitions never used

File: sme_finance_application_schema/batch_application_v1 [hunk 2]
```diff
@@ -1,9 +1,13 @@
 {
     "$schema":"http://json-schema.org/draft-04/schema#",
+    "id": "https://www.fundingoptions.com/schema/batch_application_v1",
     "title":"Finance Application Batch submission wrapper",
     "description":"A wrapper to allow multiple Finance Applications to be submitted",
     "additionalProperties": false,
     "type": "object",
+    "definitions":{
```
Comment (cybojenix) [inline sme_finance_application_schema/batch_application_v1]: never used

File: sme_finance_application_schema/finance_application_v1 [hunk 3]
```diff
@@ -1,30 +1,33 @@
...
-            "description":"A unique identifier provided by the organisation referring the applicant",
-            "type":"string"
+{
+    "$schema":"http://json-schema.org/draft-04/schema#",
+    "id": "https://www.fundingoptions.com/schema/finance_application_v1",
+    "title":"SME Finance Application wrapper",
+    "description":"A wrapper for an SME finance application and associated information identifying the applicant",
+    "additionalProperties": false,
+    "type": "object",
+    "definitions": {
```
Comment (cybojenix) [inline sme_finance_application_schema/finance_application_v1]: never used

File: sme_finance_application_schema/finance_application_v2 [hunk 4]
```diff
@@ -1,32 +1,37 @@
...
-    }
-}
+{
+    "$schema":"http://json-schema.org/draft-04/schema#",
+    "id": "https://www.fundingoptions.com/schema/finance_application_v2",
+    "title":"SME Finance Application wrapper",
+    "description":"A wrapper for an SME finance application and associated information identifying the applicant",
+    "additionalProperties": false,
+    "type": "object",
+    "definitions": {
```
Comment (cybojenix) [inline sme_finance_application_schema/finance_application_v2]: never used

File: sme_finance_application_schema/finance_application_v2 [hunk 5]
```diff
@@ -1,32 +1,37 @@
...
+    },
+    "required": [
+        "identifying_information", "finance_application", "unique_identifier"
+    ],
+    "properties": {
+        "identifying_information": {
+            "$ref": "sme_contact_v3"
+        },
+        "finance_application": {
+            "anyOf": [
```
Comment (cybojenix) [inline sme_finance_application_schema/finance_application_v2]: nit: this is technically a breaking change, however given that the original implementation was actually broken, I support this changing

File: sme_finance_application_schema/lender_v3 [hunk 6]
```diff
@@ -0,0 +1,1665 @@
+{
```
Comment (cybojenix) [inline sme_finance_application_schema/lender_v3]: I thought lender_v3/4 are no longer to be in this repo?
Especially given that we've moved away from this structure internally

File: sme_finance_application_schema/sme_v5 [hunk 7]
```diff
@@ -299,13 +299,34 @@
                 "sage_live",
                 "sage_instant",
                 "sage_line_50",
-                "sage_other", 
-                "xero_quickbooks_online",
-                "xero_quickbooks_pro",
+                "sage_other",
```
Comment (cybojenix) [inline sme_finance_application_schema/sme_v5]: discussions to happen internally (v5 is referenced in finance_application_v2)

**Comment by cybojenix** [comment]
`lender_v3/4` need removing from master branch, and the fundingoptions branch as well

**Comment by raoulsullivan** [comment]
I don't think we should be publishing out intermediate scheme (particularly sme_v5) since we never made this public.

I also wonder if we should put the python files in here at all?

**Pull request closed without merging**